modules/consensus/persist.go

```diff
@@ -25,7 +25,15 @@ func (cs *ConsensusSet) initSetDB() error {
 	if err != nil {
 		return err
 	}
-	// Explicit initilization preferred to implicit
+
+	// Update the siafundoutput diffs map for the genesis block
+	// on disk
+	cs.updateDatabase = true
+	for _, sfod := range cs.blockRoot.SiafundOutputDiffs {
+		cs.commitSiafundOutputDiff(sfod, modules.DiffApply)
+	}
```

Comment:
This is init code, that needs to happen after the genesis block is added to the consensus set and before the consensus set hash for it is computed. The function that this is in is supposed to be a first time database initialization. This loop was in consensusset.go before.